versioned_docs/version-0.6.0/features/ipam.mdx

```diff
@@ -0,0 +1,25 @@
+# IP Address Management 
+
+IP Address Management (IPAM) is a method of planning, tracking, and managing the IP address space 
+used in a network. The `maxClusters` parameter of the slice configuration represents IPAM. 
```

Comment:
it doesn't represent IPAM, it helps with it.. can we rephrase

Reply:
rephrased it as `helps with IPAM`